src/blog/2022/011-form-js-0.10.0.md

```diff
@@ -0,0 +1,141 @@
+author:
+- Roman Kostka <https://github.com/RomanKostka>
+published: 2022-12-23 18:00
+
+releases:
+  - 'form-js@0.10.0'
+
+---
+
+<p class="introduction">
+  We are excited to announce the latest [form-js@0.10 release](https://github.com/bpmn-io/form-js). 
+  This release significantly improves the capabilities of our form builder & renderer.
+</p>
+
+<!-- continue -->
+
+
+The release introduces new form components such as ‘image view’, ‘multiline text’, and ‘date time’, in order to support all basic data types and form needs. 
+It also adds the ability to add a prefix and/or suffix adornment to selected components.
+Form-js now also supports decimal numbers.
+
+Another highlight is that you are now able to hide components based on conditions and get a live preview of your form with the ability to validate it.
+
+The whole release is dedicated to helping you to build more flexible forms with awesome usability.
+Try out the new features on https://demo.bpmn.io/form/s/start.
+
+## Image View
+
+The new image view components lets you display images based on a link, data URI or variables.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-image-view.gif">
+  </a>
+
+  <p class="caption">
+    Display an image from any source using the image view component.
+  </p>
+</div>
+
+## Text View
+
+The text view component now supports FEEL expressions to display additional context via form variables and other form fields. Format your text using markdown or basic HTML.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-text-view.gif">
+  </a>
+
+  <p class="caption">
+    You can reference the value of other form components to display in the text view component.
+  </p>
+</div>
+
+## Date and Time
+
+The date and time component allows your users to display and select a date, time, or both. It is fully accessible from the keyboard.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-date-time.gif">
+  </a>
+
+  <p class="caption">
+    The new date and time picker is designed for maximum convenience, whether accessed with the mouse or keyboard.
+  </p>
+</div>
+
+## Prefix and Suffix Adornments
+
+With the new prefix and suffix adornments you are able to enrich your components with additional information, and provide guidance for your users.
+
+<div class="figure full-size">
+  <a href="https://demo.bpmn.io/form">
+    <img src="{{ assets }}/attachments/blog/2022/011-adornments.gif">
+  </a>
+
+  <p class="caption">
+    Adornments provide guidance for users in a more convenient way than labels or hints.
+  </p>
+</div>
+
+## Decimal Numbers
+
+The number component now supports decimals. You are able to define the maximum allowed decimal digit and also the increment.
```

Comment:
I think you mean the maximum number of decimal digits (or maximum precision), which is different from the maximum decimal digit. 

I think a better way to put this would be something like: You are able to define the maximum precision and valid increments of your number values.